hybrasyl/Client.cs

```diff
@@ -609,6 +625,7 @@ public void Redirect(Redirect redirect, bool isLogoff = false)
             GameLog.InfoFormat("Processing redirect");
             GlobalConnectionManifest.RegisterRedirect(this, redirect);
             GameLog.InfoFormat("Redirect: cid {0}", this.ConnectionId);
+            GameLog.InfoFormat($"Redirect EncryptionKey is {Encoding.ASCII.GetString(redirect.EncryptionKey)}");

```

Comment:
same here

Reply:
i left it info currently so it's easy to identify

Reply:
🤷‍♂